website/docs/docs/build/snapshots.md

```diff
@@ -587,6 +587,16 @@ Snapshot results:
 
 </details>
 
+<VersionBlock firstVersion="1.9">
+
+### Define column names in snapshot table
+
```

Comment:
small sugg that's not a blocker: maybe have the header outside the versionblock and tell users on 1.8 and lower that this feature is only avail in 1.9 and higher or versionless so they know?

Reply:
++ we need to do this for all new features!